### ISSUE TITLE
Add Parallax CSS style.

### ISSUE THREAD

**Issue by honeysilvas** [blastpad/BlastPad_UI#4]
Add Parallax CSS style.

This is a good first bug and can be a mentored bug.  Please message me here if you need more information.

**Comment by Shaun2D2** [comment]
@honeysilvas ,

Looking to contribute to Hacktoberfest and I saw your project on myfirstpr.  Neat little CSS framework, i'm in to helping out!

Let me know what you need done here and I'll see what I can do to help! :D

Shaun

**Comment by honeysilvas** [comment]
Hi, @Shaun2D2 

Sounds great!  

If you haven't done so yet, you can check out our guide to contributing at https://github.com/blastpad/BlastPad_UI/wiki/Contributing

Take a look at our Issue Tracker at https://github.com/blastpad/BlastPad_UI/issues for current issues.  If you see an issue you like, just dive in.  I can give you more information about the issue.  

If you want to do this specific issue (#4 Add Parallax CSS style), I can assign that to you.

You can also add issues to the issue tracker if you see anything that needs to be done.

If you have time, I would be happy to chat with you via Slack.  Just let me know what time would work for you and we can set it up.

**Comment by acdpsn** [comment]
Hi, is this issue still open for fixing? I'd like to take a crack at it for Hacktoberfest if it is.

**Comment by honeysilvas** [comment]
@akim8 

Yes, this is still open.  You can work on this.   Please let me know if you have any questions or need help.

Thanks!
